backend/storage/schema.sql

```diff
@@ -229,3 +229,18 @@ CREATE TABLE wallets (
 );
 
 CREATE INDEX wallets_by_account ON wallets (account);
+
+-- Stores text content to to a full text search
+-- https://sqlite.org/fts5.html.
+
+CREATE VIRTUAL TABLE fts USING fts5(
+    -- The text content to be indexed.
+    raw_content, 
+    -- The type of the content being indexed.
+    type, 
```

Comment:
@juligasa What does type mean here? Can you document what possible types are supported here?